packages/cozy-client/src/models/document/DocumentTypeData.js

```diff
@@ -0,0 +1,211 @@
+import { Qualification } from './qualification'
```

Comment:
I find it weird to have `DocumentTypeData`, `DocumentTypeDataHelpers` with the first letter in uppercase, while `qualification.js` is lowercase. I think the overall style is to have capital letters when the default export is a class, so we might want to have `documentTypeData` and `documentTypeDataHelpers` ?

Reply:
Indeed, it makes more sense